src/attr.rs

```diff
@@ -350,11 +350,34 @@ impl Rewrite for ast::Attribute {
             if contains_comment(snippet) {
                 return Some(snippet.to_owned());
             }
+
+            let meta = self.meta();
```

Comment:
nit: may as well inline this variable

Reply:
Actually, maybe pull the `if let` to the top level, then you can shuffle the code around to move the body of the `and_then` from line 380 into the `if let`